modules/deploy/pages/deployment-option/self-hosted/docker-image.adoc

```diff
@@ -3,7 +3,7 @@
 :page-context-links: [{"name": "Docker", "to": "deploy:deployment-option/self-hosted/docker-image.adoc" },{"name": "Kubernetes", "to": "deploy:deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc" } ]
 :page-toclevels: 1
 
-This topic describes how to use the https://hub.docker.com/r/redpandadata/connectors/tags[Redpanda Connectors Docker image^] to configure and deploy connectors for Redpanda. The image contains:
+This topic describes how to use the https://hub.docker.com/r/redpandadata/connectors/tags[Redpanda Connectors Docker image^] to configure and deploy connectors for Redpanda. This configuration requires you to configure and use https://redpanda.com/guides/kafka-tutorial/what-is-kafka-connect[Kafka Connect]. https://docs.redpanda.com/current/deploy/deployment-option/cloud/managed-connectors/[Managed Connectors] are available only in Redpanda Cloud. The Docker image contains:
```

Comment:
**Redpanda Connector Docker image is a community supported artefact.** 

This topic describes how to use the [Redpanda Connectors Docker image](https://hub.docker.com/r/redpandadata/connectors/tags). 

Redpanda Connectors Docker image contains [Kafka Connect](https://redpanda.com/guides/kafka-tutorial/what-is-kafka-connect) pre-configured to work with Redpanda. Kafka Connect is an Apache tool to reliably stream data between Redpanda/Kafka and other data systems. 

Redpanda Connectors Docker image contains only the MirrorMaker2 connector. Please see the instructions on how to install more connectors.

For smoother experience, consider using the [Managed Connectors](https://docs.redpanda.com/current/deploy/deployment-option/cloud/managed-connectors/) in Redpanda Cloud, available for instant use.

Reply:
@vladoschreiner If the takeaway that this is community-supported is that it is not supported by RP, shouldn't we tell people to use our community Slack channel for support and provide a link? I've taken the liberty of doing so. Also, given that we link to a description of Apache Connect, I thought the additional verbiage there was superfluous. Please let me know if this updated version is acceptable. Thanks!